stage1_fly/run/main.go

```diff
@@ -334,7 +334,7 @@ func stage1() int {
 		}
 	}
 
-	if err = stage1common.WritePid(os.Getpid(), "ppid"); err != nil {
+	if err = stage1common.WritePid(os.Getpid(), "pid"); err != nil {
```

Comment:
This change looks good to me.

But be aware of https://github.com/coreos/rkt/pull/2438/files#r67318524 :)

/cc @jjlakis